tests/osautoinst/test_running.pm

```diff
@@ -4,7 +4,7 @@ use testapi;
 
 sub run {
     assert_script_run 'command -v ack >/dev/null || zypper --no-refresh -n in ack';
-    assert_script_run 'ret=false; for i in {1..5} ; do openqa-client jobs state=running | ack --passthru --color running && ret=true && break ; sleep 30 ; done ; [ "$ret" = "true" ]', 300;
+    assert_script_run 'ret=false; for i in {1..10} ; do openqa-client jobs state=running | ack --passthru --color running && ret=true && break ; sleep 30 ; done ; [ "$ret" = "true" ]', 300;
```

Comment:
10*30 plus overhead exceeds the timeout of 300s. Better increase that as well then if you really want to bump the retries

Reply:
No I don't want to, that's why I converted it to a **draft**